src/core/app_messages.tf

```
@@ -0,0 +1,290 @@
+resource "azurerm_resource_group" "io-p-app-messages-rg-01" {
+  count    = var.app_messages_enabled ? 1 : 0
+  name     = format("%s-io-p-app-messages-rg-01", local.project)
```

```suggestion
  name     = format("%s-app-messages-rg-01", local.project)
```
local project contains `io-p-` prefix